### PR TITLE
Panics in script execution are handled

### PR DESCRIPTION
**Pull request by m4ksio**

### CODE REVIEW

File: engine/execution/computation/manager_test.go [hunk 1]
```diff
@@ -139,3 +141,38 @@ func TestExecuteScript(t *testing.T) {
...
+	header := unittest.BlockHeaderFixture()
+
+	manager, err := New(log, nil, nil, nil, nil, vm, ctx)
+	require.NoError(t, err)
+
+	_, err = manager.ExecuteScript([]byte("whatever"), nil, &header, view)
+
+	require.Error(t, err)
+
+	require.Contains(t, buffer.String(), "Verunsicherung")
```
Comment (turbolent) [inline engine/execution/computation/manager_test.go:166]: 😆

File: engine/execution/computation/manager.go [hunk 2]
```diff
@@ -84,7 +86,27 @@ func (e *Manager) ExecuteScript(code []byte, arguments [][]byte, blockHeader *fl
...
+				err = fmt.Errorf("cadence runtime error: %s", r)
+
+				args := make([]string, 0)
+				for _, a := range arguments {
+					args = append(args, hex.EncodeToString(a))
+				}
+				e.log.Error().
+					Hex("script_hex", code).
+					Str("args", strings.Join(args[:], ",")).
+					Interface("r", r).
```
Comment (turbolent) [inline engine/execution/computation/manager.go:102]:
```suggestion
					Interface("recovered", r).
```

File: engine/execution/computation/manager.go [hunk 3]
```diff
@@ -84,7 +86,27 @@ func (e *Manager) ExecuteScript(code []byte, arguments [][]byte, blockHeader *fl
...
 	script := fvm.Script(code).WithArguments(arguments...)
 
-	err := e.vm.Run(blockCtx, script, view)
+	err := func() (err error) {
+
+		defer func() {
+			if r := recover(); r != nil {
+				err = fmt.Errorf("cadence runtime error: %s", r)
+
+				args := make([]string, 0)
```
Comment (turbolent) [inline engine/execution/computation/manager.go:95]:
```suggestion
				args := make([]string, 0, len(arguments))
```

**Review by turbolent — APPROVED** [review]
Nice! Thanks for adding this

**Review by Kay-Zee — APPROVED** [review]

**Pull request merged**